lib/developer_portal/app/views/developer_portal/javascripts/3scale_v2.js

```diff
@@ -286,7 +286,7 @@ var hljs=new function(){function e(e){return e.replace(/&/gm,"&amp;").replace(/<
       return rails.blankInputs(form, specifiedSelector, true); // true specifies nonBlank
     },
 
-    // Helper function, needed to provide consistent behavior in IE
+    // Helper function
```

Comment:
I am a little confused with the comment deleted, I guess we still need this helper function but not specifically for IE, right? 😄

Reply:
Good question.

Reply:
No, I don't think we do. Suggestions for refactoring it welcome!

Reply:
I think question is whether we need to delete this function now.

Reply:
I don't think we need this helper function. I couldn't come up with a way to refactor de code in order to delete it.

Reply:
The whole code block seems to be coming from smth called jquery_ujs: https://github.com/rails/jquery-ujs/blob/4bdd8724d41a4dc84aa3f9b29381adb47e6ee215/src/rails.js#L361

I'd keep both the code and the comment, it does not do any harm

Reply:
Agreed, if IE specific code is not removed, then comment should stay. Otherwise somebody will be baffled at to why is this code like that instead of something else.